arcgis-ios-sdk-samples/Maps/Download preplanned map/PreplannedMapAreaTableViewCell.swift

```diff
@@ -0,0 +1,29 @@
+//
```

Comment:
Need to update header to the common header we use in all samples

Reply:
Pardon the oversight. I tidied this up via [36cb6277](https://github.com/Esri/arcgis-runtime-samples-ios/pull/758/commits/36cb627778df095b758f79a1019c23723b73ae91).